### PR TITLE
Page-wise progress bars

### PR DESCRIPTION
**Pull request by adswa**
After #193 I looked a bit into "scrolling progress bars". This isn't yet what @lisanmo suggests (it's only page-wise) and I'm not sure yet whether or how one could get a chapter-wise progress bar in the HTML version of the book, but I will continue exploring. This is how it currently looks like (The blue bar progresses the more you scroll):

image

### CODE REVIEW

File: docs/_templates/sidebarlogo.html [hunk 1]
```diff
@@ -20,3 +20,5 @@
   height: 100%;
 }
 </style>
+
+<progress id="progressbar" value="0" max="100"></progress>
```
Comment (adswa) [inline docs/_templates/sidebarlogo.html]: brief thought to follow up later: Maybe its possible to add this progress bar page-wise into the header, and set ``value`` and ``max`` based on a counter in each chapter. A chapter with 5 sections would add one fifth to the progress bar per page read (while pace of progress bar for that fifth is still dependent on section length).

**Comment by adswa** [comment]
Having played with this a bit more, I failed to implement a progress bar that spans chapters. It may well be possible, but for now I'm giving up. As it currently is, this PR would add a page-wise progress bar. IMO it's a nice-to-have but not a need-to-have.

**Comment by mih** [comment]
+1

**Comment by adswa** [comment]
(if nobody complains within the next day or so I will let us have page-wise progress bars)

**Pull request merged**